src/vanilla/Model/CodeModelTS.cs

```diff
@@ -811,9 +812,9 @@ public virtual bool HasMappers()
         public virtual string GenerateMapperIndex()
         {
             TSBuilder builder = new TSBuilder();
-            builder.Comment(AutoRest.Core.Settings.Instance.Header);
+            builder.Line(LicenseHeader.GenerateLicenseHeader());
             builder.Line();
-
+            
```

Comment:
nit: please remove these extra spaces

Reply:
Sorry about that! I have my editor set up to remove extra white space. Not sure why it missed this one. Fixed.